dev-docs/source/MVPTutorial/CompleteGUI.rst

```diff
@@ -31,19 +33,19 @@ Main module
 
             self.window = QtGui.QMainWindow()
 
-            my_view = masterView.MasterView(parent=self)
-            self.master_presenter = masterPresenter.MasterPresenter(my_view, data_model, colour_model)
+            my_view = master_view.MasterView(parent=self)
+            self.master_presenter = master_presenter.MasterPresenter(my_view, data_model, colour_model)
 
             # set the view for the main window
             self.setCentralWidget(my_view)
             self.setWindowTitle("view tutorial")
 
-        def qapp():
-            if QtGui.QApplication.instance():
-                _app = QtGui.QApplication.instance()
-            else:
-                _app = QtGui.QApplication(sys.argv)
-            return _app
+    def qapp():
+        if QtGui.QApplication.instance():
+            _app = QtGui.QApplication.instance()
```

Comment:
```suggestion
            app = QtGui.QApplication.instance()
```

There is no technical reason for this underscore to be here so we should not suggest there being one by including it.

Reply:
I've gone through and gotten rid of any different capitalizations that don't reffer to classes.